spec.bs

```diff
@@ -1213,7 +1218,7 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
 
 <div algorithm>
   To <dfn>evaluate a bidding script</dfn> given a [=string=] |script|, an [=interest group=] |ig|,
-  and a [=list=] of arguments |arguments|:
+  a [=list=] of arguments |arguments|, and an integer |timeout|:

```

Comment:
maybe we can make this more clear by saying a duration specified in milliseconds?